test/index.test.js

```diff
@@ -334,5 +356,68 @@ describe('module-resolver', () => {
                 transformerOpts,
             );
     });
+
+    describe('unknown filename', () => {
```

Comment:
What is unknown here?

From what I understand, this is just a test to make sure the `babelrc` cwd works as expected?